docs/reference/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -83,15 +86,28 @@ It supports nested data structure.
 Within the `metadata` object, keys beginning with `_` are reserved for
 system usage.
 
+`client_authentication`::
+(Optional, object) When using the `access_token` grant type, and when supplying a
+JWT, this specifies the client authentication for <<JWT access tokens, jwt-realm-oauth2>>
```

Comment:
> jwt-realm-oauth2

is that link right ? That link looks like it is only for JWT access tokens, while this is applicable for both access and id tokens. Is there a better link we can use here (or just omit the link).

Reply:
I've changed it to the JWT authentication top of the page https://www.elastic.co/guide/en/elasticsearch/reference/current/jwt-auth-realm.html#jwt-auth-realm with `jwt-auth-realm` .